test/input/experiments/ipnsw.gt

```diff
@@ -0,0 +1,146 @@
+element Vertex end
+element Edge end
+const edges : edgeset{Edge}(Vertex,Vertex);
+const vertices : vertexset{Vertex};
+const graphs : vector[3](edgeset{Edge}(Vertex, Vertex));
+const data: vector{Vertex}(vector[100](float));
+const ef: int;
+const v_entry: int;
+const qs: vector[512](vector[100](float));
+
+const pqs: vector[512](priority_queue{Vertex}(pair?));
+const topk_pqs: vector[512](priority_queue{Vertex}(pair?));
+
+var d_currs: vector[512](float) = 0.0;
+var v_currs: vector[512](int) =0;
+
+export func set_up(edges_arg: edgeset{Edge}(Vertex, Vertex),
+                   graphs_arg : vector[3](edgeset{Edge}(Vertex, Vertex),
+                   data_arg : vector{Vertex}(vector[100](float)),
+                   ef_arg : int,
+                   v_entry_arg : int)
+
+       edges = edges_arg;
+       graphs = graphs_arg;
+       vertices = edges.getVertices();
+       data =  data_arg;
+       ef = ef_arg;
+       v_entry = v_entry_arg;
+end
+
+
+func not_seen<cur_seen>(v:Vertex) -> output : bool
+     if (cur_seen[v])
+         output = false;
+     else
+         output = true;
+     end
+end
+
+
+func update_distance_vertex<dist: float, node: Vertex, q: vector[100](float)>(v: Vertex) -> output: bool
+
+    var new_dist: float = dist(data[v], q);
+    if dist < new_dist
+        dist = new_dist;
+        node = v;
+        output = true;
+    else
+        output = false;
+    end
+
+end
+
+
+func update_edge<seen: vector{Vertex}(bool), q: vector[100](float), pq: priority_queue, topk_pq: priority_queue>(src: Vertex, dest: Vertex)
```

Comment:
The syntax here doesn't match rest of the test cases/parser. Instead of passing the topk_pq can we pass just an index? Since topk_pqs is global.

Reply:
will fix that. thanks